### PR TITLE
Fix move enable-api-fields to feature-flags CM

### PR DESCRIPTION
**Pull request by vinamra28**
# Changes

With PR #411 the enable-api-fields for triggers was set in
config-defaults-triggers ConfigMap whereas it should be added in
feature-flags-triggers ConfigMap so shifting that.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [X] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes
```release-note
NONE
```

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]

**Review by pradeepitm12 — COMMENTED** [review]
/lgtm

File: pkg/apis/operator/v1alpha1/tektontrigger_types.go [hunk 1]
```diff
@@ -82,6 +82,13 @@ type Trigger struct {
 // TriggersProperties defines the fields which are to be
 // defined for triggers only if user pass them
 type TriggersProperties struct {
+	EnableApiFields string `json:"enable-api-fields,omitempty"`
+	// +optional
+	OptionalTriggersProperties `json:",inline"`
```
Comment (sm43) [inline pkg/apis/operator/v1alpha1/tektontrigger_types.go:87]: 👍🏽

**Pull request merged**